cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -672,14 +676,7 @@ impl pallet_xcm::Config for Runtime {
 	type XcmExecuteFilter = Nothing;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 	type XcmTeleportFilter = Everything;
-	// Allow reserve based transfer to everywhere except for bridging, here we strictly check what
-	// assets are allowed.
-	type XcmReserveTransferFilter = (
-		LocationWithAssetFilters<WithParentsZeroOrOne, AllAssets>,
-		bridging::to_rococo::AllowedReserveTransferAssets,
-		bridging::to_wococo::AllowedReserveTransferAssets,
-	);
-
+	type XcmReserveTransferFilter = Everything;
```

Comment:
Just want to confirm - is that an intended change? Why we are not checking it anymore? Because it is a test bridge and we want to test multiple asset transfers there?

Reply:
this filter was wrong, original intention was to filter `(dest, asset)` - to allow particular asset to be sent to particular destination. But this `XcmReserveTransferFilter` or `XcmTeleportFilter` takes `origin` and not `destination` as I mistakenly thought. We call this problem "shoot oneself in the foot" - actually, `pallet_xcm` does not support fitlers like "what assets are sent to what destination".


@franciscoaguirre Cisco, I think it would be cool to udpate docs (beyond this PR) for:
```
                /// Our XCM filter which messages to be teleported using the dedicated extrinsic must pass.
		type XcmTeleportFilter: Contains<(MultiLocation, Vec<MultiAsset>)>;

		/// Our XCM filter which messages to be reserve-transferred using the dedicated extrinsic
		/// must pass.
		type XcmReserveTransferFilter: Contains<(MultiLocation, Vec<MultiAsset>)>;
```
It is not clear what `XCM filter` and `MultiLocation` mean in the context of filtering here.